File: tkpdinterface/src/main/java/com/tokopedia/interfaces/merchant/shop/info/ShopInfoBadgeLevel.java

package com.tokopedia.interfaces.merchant.shop.info;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class ShopInfoBadgeLevel {

    @SerializedName("level")
    @Expose
    private long level;
    @SerializedName("set")
    @Expose
    private long set;

    public long getLevel() {
        return level;
    }

    public void setLevel(long level) {
        this.level = level;
    }

    public long getSet() {
        return set;
    }

    public void setSet(long set) {
        this.set = set;
    }

}

File: features/merchant/shop_common/src/main/java/com/tokopedia/shop/common/data/source/cloud/model/ShopInfoBadgeLevel.kt
package com.tokopedia.shop.common.data.source.cloud.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class ShopInfoBadgeLevel {
    @SerializedName("level")
    @Expose
    var level: Long = 0

    @SerializedName("set")
    @Expose
    var set: Long = 0
}